src/actions/run-command.h

```diff
@@ -39,6 +39,8 @@ class RunCommand : public RepeatedAction {
   std::string decreaseCommand;
 
   static bool runCommand(const std::string &command);
```

Comment:
This method is private and only invoked by 2 methods. I think we should delete it instead of overloading it.